src/lib/common/Kconfig

```diff
@@ -32,6 +32,9 @@ config DETECT_BOARD_NAME
 config PLATFORM_LINUX
 	bool
 
+config PLATFORM_CONTIKI_QMSI
```

Comment:
On Thu, 19 May 2016 06:53:16 -0700, Gustavo Lima Chaves wrote:
> > @@ -32,6 +32,9 @@ config DETECT_BOARD_NAME
> >  config PLATFORM_LINUX
> >  	bool
> >  
> > +config PLATFORM_CONTIKI_QMSI
> 
> Cool, but why not place it near the other Contiki platform declaration?

Because that one is inside a choice block.